etc/ReplayOfflineConfiguration.py

```diff
@@ -1131,6 +1130,18 @@
 ignoreStream(tier0Config, "streamDQMRates")
 ignoreStream(tier0Config, "DQMPPSRandom")
 
+# Run only on the ZB streams
+specifyStreams(tier0Config, "PhysicsZeroBias0")
+specifyStreams(tier0Config, "PhysicsZeroBias1")
+specifyStreams(tier0Config, "PhysicsZeroBias2")
+specifyStreams(tier0Config, "PhysicsZeroBias3")
+specifyStreams(tier0Config, "PhysicsZeroBias4")
+specifyStreams(tier0Config, "PhysicsZeroBias5")
+specifyStreams(tier0Config, "PhysicsZeroBias6")
+specifyStreams(tier0Config, "PhysicsZeroBias7")
+specifyStreams(tier0Config, "PhysicsZeroBias8")
+specifyStreams(tier0Config, "PhysicsZeroBias9")
```

Comment:
Please, add them in a single call as a list:
```suggestion
specifyStreams(tier0Config, ["PhysicsZeroBias0", "PhysicsZeroBias1", "PhysicsZeroBias2", "PhysicsZeroBias3", "PhysicsZeroBias4", "PhysicsZeroBias5", "PhysicsZeroBias6", "PhysicsZeroBias7", "PhysicsZeroBias8", "PhysicsZeroBias9"])
```

Reply:
I know it is inconsistent on how `ignoreStream` works. But we prefer this behaviour, and will be changing `ignoreStream` to work like this too.

Reply:
ok no problem! I'll do the update in few minutes!

Reply:
done in [45e8450](https://github.com/dmwm/T0/pull/4817/commits/45e84508d5275e7794fbb488b2dce330fc1c3d06)